src/Form/ChoiceList/ModelChoiceLoader.php

```diff
@@ -52,40 +55,27 @@ final class ModelChoiceLoader implements ChoiceLoaderInterface
     private $query;
 
     /**
-     * @var object[]
+     * @var object[]|null
```

Comment:
Are you sure about the `null` value ?
It's supposed to be set by the constructor.

Reply:
Fixed